[REF] marketing automation: move getting started

samueljlieber opened the pull request:
Task: [#3699212](https://www.odoo.com/web#id=3699212&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

This PR restructures the Marketing Automation directory to remove the `getting_started/*` sub-directory. This restructure is laying the foundation for future improvements to this app's documentation.

```
getting_started.rst --> marketing_automation.rst
getting_started/first_campaign.rst --> marketing_automation.rst
getting_started/* --> marketing_automation/*
getting_started/* --> marketing_automation/*
getting_started/* --> marketing_automation/*
```

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/7658).

samueljlieber commented:
Updated :doc: path on `testing_running.rst` in https://github.com/odoo/documentation/commit/ff38935ac49d454d88e41e7e2b41c4221e2ebdd7

samueljlieber commented:
> Just a quick thing— the toctree isn't rendering correctly on `marketing_automation.rst`. Can you take a look?

Hi @StraubCreative, thank you for the review! After some testing, this is how the Page TOC formats when there is a subdirectory below it, and not just individual files. So in this case the `marketing_automation/advanced/understanding_metrics` line is causing the TOC to format like this. I agree it looks kinda off! I am swapping to use a seealso since it is a little cleaner 👍
<img width="355" alt="Screenshot 2024-02-07 at 9 29 52 AM" src="https://github.com/odoo/documentation/assets/36018073/fee9b93e-b2cd-452c-9a83-be9442d69d10">

StraubCreative commented:
> After some testing, this is how the Page TOC formats when there is a subdirectory below it

Gross 😅 We should change it + touch up the styles anyway, they could use some love.

StraubCreative commented on content/applications/marketing/marketing_automation.rst:
```diff
@@ -1,17 +1,126 @@
+   marketing_automation/target_audience
+   marketing_automation/workflow_activities
+   marketing_automation/testing_running
    marketing_automation/advanced
```
Do we think having a sub-directory called `advanced` is necessary?
Why not just have it at the same level as all the other docs as this also solves the toctree display issue?

What do you think?

samueljlieber replied:
I agree, my plans for improving this documentation do not invlove an `/advanced` directory, so I am happy to remove this as well and move `understanding_metrics.rst` to the same level as the others 👍

samueljlieber commented:
In https://github.com/odoo/documentation/commit/22fb8d17a18b2b6c0a7e56edffe4ba2bfbfe09be I removed the `/advanced` directory and moved `understanding_metrics.rst` to the same level as the other docs.

StraubCreative reviewed (approved):
Seems good, thanks Sam 👍 

@robodoo r+

robodoo commented:
@samueljlieber @StraubCreative staging failed: ci/runbot on 881707cdef571579ec8e69fb16cfa3a7130fe634 (view more at https://runbot.odoo.com/runbot/build/57947793)

StraubCreative commented:
@robodoo retry

robodoo commented:
@samueljlieber @StraubCreative staging failed: ci/runbot on 6d0ec28df549613860407851ea61ccb7a191d745 (view more at https://runbot.odoo.com/runbot/build/57948362)

samueljlieber commented:
@robodoo retry
